.github/workflows/sync-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: sync-tests
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - '*rc'
+      - 'regenesis/*'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  integration-sync-test:
+    runs-on: ubuntu-latest
+    env:
+        DOCKER_BUILDKIT: 1
+        COMPOSE_DOCKER_CLI_BUILD: 1
+    steps:
+      # Monorepo tests
+      - uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Build the services
+        working-directory: ./ops
+        run: ./scripts/build-ci.sh
+
+      - name: Bring the stack up
+        working-directory: ./ops
+        run: docker-compose up -d && ./scripts/wait-for-sequencer.sh
+      
+      - name: Run the sync tests
+        working-directory: ./integration-tests
+        run: |
+          yarn
+          yarn test:sync
```

Comment:
Is there any reason that this is a different workflow than the other integration tests? Ideally the images are built once and then used twice in parallel

Reply:
^hm i can look into this! just know we need a fresh sequencer for this set of tests

Reply:
I had started working on that [here](https://github.com/ethereum-optimism/optimism/pull/891/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bR41), agreed with Mark but I think it's better to do this in another PR, in the interest of saving Annie's time

Tl;dr: You'd have 1 job which builds the services, saves the docker images to a tar and then uploads the artifacts. Then in the tests job you'd download the artifacts, load the docker images and run the tests.